Mobile-device-testing.md

```diff
@@ -22,6 +22,13 @@ A lot of developers have faced an issue "Request Host not whitelisted". To worka
 1. Once the simulator app icon appears in the dock, you can either pin it in the dock, or move/copy the app to a more convenient location for future access. This allows you to open the simulator without having to first open Xcode.
 1. Open Safari, then navigate to a locally-hosted developer instance to test Oppia on iOS.
 
+#### Directly run the simulator via command line
+
+1. See a list of available simulators using `xcrun simctl list`.
+Example output: `iPhone 11 (17DDCE3C-1V29-4251-BC19-21168CA1B259) (Shutdown)`
+The string of alphanumerics within the `()` is the UDID.
+1. Run a particular device using its UDID `open -a Simulator --args -CurrentDeviceUDID <device UDID>`.
```

Comment:
Please edit the numbers to make them sequential.

Reply:
They will be displayed sequentially. This is just for cleanliness and not having to readjust a lot of numbers if a new point is added somewhere in the middle. A similar convention is followed elsewhere in the codebase too.

Reply:
Changing the numbers to make them sequential would improve readability in raw markdown. This is done to prevent confusion about whether 2 points are intended to come one after another or are separate ones.

Reply:
I understand but the same convention is followed elsewhere in this codebase. Should we then change it everywhere?

Reply:
Just change it for this file. I'll create an issue to resolve the remaining files. Thanks for bringing it up!